Allow deploying multiple tiles on multiple cf instances info

okilker opened the pull request:
Added information to: appdynamics-platform/installing.html, may need an h2 headline?
Wasn't sure if this information should live at the bottom of this appdynamics-platform/using.html instead?

jncd commented:
Hi @okilker 
I've downloaded and done a local build to look at your PR.
Do you have a tech writer on staff who you could work with on this section?
It doesn't look good enough to merge yet. Here are some points to consider before resubmitting:

 - Is this section really a subsection of Advanced Configuration of an AppDynamics Application? If yes, then restructure that h2 section to contain two h3s, etc. If no, then make it an h2.

- Stem sentence to list doesn’t make logical sense to me: How can _reviewing_ a scenario _implement_ a change?

- Define a “cf instance” — you only use instance in one other place in the docs, in the FAQ: https://docs.pivotal.io/partners/appdynamics-platform/faqs.html#d5. 

- Who are the customers? This is the only use of the word in this doc. 

- You’ve used a numbered list but I don’t see that the sequence of the items is important, it’s not a procedure or a process.

- How is a CF foundation different from a PCF foundation — do you really mean a CF foundation is an open source deployment from the Cloud Foundry Foundation without Ops Manager or any UI? Or do you mean a regular Pivotal Cloud Foundry deployment that had Ops Manager and a deployed tile?

- The PR has style issues:
    - Capitalization of the heading
    - Punctuation
    - Grammar

- Always add the `<a id='some-anchor-text'></a>` anchor code to h2s and h3s.

We'll look forward to merging a better version of the PR in 2019! Happy New Year.

Jane

okilker commented:
Hi @jncd, thanks so much for your thorough review! I worked with Pavan to simplify the paragraph. Please let us know if this version addresses your concerns. Thanks and Happy New Year!
